### PR TITLE
Add missing client info schema to the app User section

### PR DESCRIPTION
**Pull request by Go-Fred**
The client info schema was not documented. This PR adds it to the API docs and takes into account the different channels where the fields are present.

### CODE REVIEW

**Comment by mspensieri** [comment]
I think it was intended to not document this schema: 

https://github.com/smooch/smooch-docs/pull/278#discussion_r105492539
https://github.com/smooch/smooch-docs/pull/278/commits/6e681d057869ff947d6d8db57b0621390e9b13d4
https://github.com/smooch/smooch-docs/pull/278#discussion_r105948140

**Comment by Go-Fred** [comment]
We're asking this question by customers often. So, we wanted to do it: https://trello.com/c/yWtvpwIf/209-docs-api-reference-document-client-info-properties

**Comment by Go-Fred** [comment]
I'm open to discuss it with the team :)

File: slate/source/includes/rest/core/_app_user.md [hunk 1]
```diff
@@ -598,7 +598,44 @@ Client specific info
...
+| **os** <span class="opt">optional</span>|    The OS of the AppUser device. Example: `"Android"`  | iOS, Android |
+| **osVersion** <span class="opt">optional</span>|   The OS version of the AppUser device. Example: `"6.0"` | iOS, Android |
+| **appName** <span class="opt">optional</span>|  For the SDK in a native app, signifies the name of the app. | iOS, Android |
+| **installer** <span class="opt">optional</span>|  The appUser device's installer.| iOS, Android |
+| **devicePlatform** <span class="opt">optional</span>|  The AppUser device platform. ex:`"x86_64"`.| iOS, Android |
+| **carrier** <span class="opt">optional</span>|  The AppUser device carrier. Take `"nil"` if none is specified. | iOS, Android |
+| **appId** <span class="opt">optional</span>|   For the SDK in a native app, signifies the appId. | iOS, Android |
+| **radioAccessTechnology** <span class="opt">optional</span>         | String that specifies the radio access technology used by the AppUser device. Take `"nil"` if none is specified. | iOS, Android |
+| **buildNumber** <span class="opt">optional</span>|   The build Number of the iOS app on which the SDK is initialized.| iOS |
```
Comment (spasiu) [inline slate/source/includes/rest/core/_app_user.md]: channels should match the way they're represented in the API, so "Facebook" should be "messenger", etc.

Comment (Go-Fred) [inline slate/source/includes/rest/core/_app_user.md]: Thanks, didn't see your answer before :(

**Comment by jpjoyal** [comment]
Let's hold a bit on this PR and repurpose it with these upcoming enhancements https://trello.com/c/7exn48MW/2073-api-expose-raw-client-info-for-ott-channels

**Comment by Go-Fred** [comment]
Okay! Let me know!

**Comment by spasiu** [comment]
Going to hold off on client info until we implement raw props.

**Pull request closed without merging**